### PR TITLE
KOGITO-4431 - Fix native build pom

### PR DESCRIPTION
**Pull request by r00ta**
JIRA: https://issues.redhat.com/browse/KOGITO-4431

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -264,25 +265,6 @@
...
-            <artifactId>quarkus-maven-plugin</artifactId>
-            <executions>
-              <execution>
-                <configuration>
-                  <enableHttpUrlHandler>true</enableHttpUrlHandler>
-                  <enableHttpsUrlHandler>true</enableHttpsUrlHandler>
-                  <enableJni>true</enableJni>
-                  <enableAllSecurityServices>true</enableAllSecurityServices>
-                  <additionalBuildArgs>--allow-incomplete-classpath</additionalBuildArgs>
-                </configuration>
```
Comment (danielezonca) [inline pom.xml]: @r00ta 
Have you tested with native compilation? I'm not sure it is safe to remove all the these other properties too

Comment (r00ta) [reply pom.xml:280]: @danielezonca Yes, tests are green

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by MarianMacik** [comment]
I have fixed the native build in #632.

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Jenkins run native

**Comment by r00ta** [comment]
Not needed anymore since #632 has fixed this

**Pull request closed without merging**